### PR TITLE
TK245- dropdown width issue solved

### PR DESCRIPTION
**Pull request by karuna835**

### CODE REVIEW

File: b/views/user.html [hunk 1]
```diff
@@ -843,7 +843,7 @@ <h5 class="modal-title">{{tempUser.name}}'s Profile</h5>
 									</select>
 								</div>
 								<div class="form-group bmd-form-group w-45 ml-auto">
-									<select class="form-control select2-user" select2=""  id="specializationBatch" ng-model="editableSubs.batch" ng-change="batchChange()" ng-attr-disabled="{{!editableSubs.editable ? 'true' : 'false'}}">
+									<select class="form-control select2-user" select2=""  id="specializationBatch" ng-model="editableSubs.batch" ng-change="batchChange()" ng-attr-disabled="{{!editableSubs.editable ? 'true' : 'false'}}"style="width:100% !important">
```
Comment (kailasb) [inline b/views/user.html:846]: add space before style tag

**Review by kailasb — CHANGES_REQUESTED** [review]
check feedback

File: b/views/user.html (hunk 1 above)
Comment (karuna835) [reply b/views/user.html]: done

**Review by kailasb — APPROVED** [review]
changes are proper

**Pull request merged**